### PR TITLE
Fixes minor issues with shadekin

### PR DESCRIPTION
**Pull request by Heroman3003**
Fixes spelling in one of ability descriptions.

Adds infinite energy var to shadekin brain.

Potentially, more to come as testing happens

### CODE REVIEW

File: code/modules/organs/subtypes/shadekin.dm [hunk 1]
```diff
@@ -2,4 +2,5 @@
 	can_assist = FALSE
 
 	var/dark_energy = 100
-	var/max_dark_energy = 100
\ No newline at end of file
+	var/max_dark_energy = 100
+	var/dark_energy_infinite = FALSE
```
Comment (KasparoVy) [inline code/modules/organs/subtypes/shadekin.dm]: Badass var name.

Comment (Heroman3003) [reply code/modules/organs/subtypes/shadekin.dm]: infinity_unlimited_flame_of_incomprehensible_dark_beyond was a bit of a mouthful

Comment (KasparoVy) [reply code/modules/organs/subtypes/shadekin.dm]: Could always acronym it!

Comment (TheFurryFeline) [reply code/modules/organs/subtypes/shadekin.dm]: :joy_cat:

**Pull request merged**